hci/command/commands/hci_ext_packet_error_rate_cmd.py

```diff
@@ -0,0 +1,44 @@
+from struct import pack, unpack
+
+from ..command_packet import CommandPacket
+from enum import IntEnum
```

Comment:
`enum` should be imported with the top group of imports.
First group contains python modules, second group contains project modules.